### DISCUSSION TITLE
Using Ansible Module directly as a template

### DISCUSSION THREAD

**Discussion by sparsick** [semaphoreui/semaphore#1210] [Q&A] [answered]
First, thanks for this nice project.

Currently, I'm evaluating it for one of my project. One question comes up. Is it possible to create a template that run an Ansible module directly, like I would do it via CLI ?

For example:

````
 ansible application_server -i inventories/test -m ping     
````

**Comment by ansibleguy** [comment, marked as answer]
Greetings.

As I see it - Semaphore currently can only run Playbooks.
If you can achieve it using a prepared playbook - the answer should be yes. (a playbook with the only task in it being a call to 'ansible.builtin.ping')

But ad-hoc commands seem to not be supported (yet).

\- AnsibleGuy
